### ISSUE TITLE
First name, last name, e-mail address not translated

### ISSUE THREAD

**Issue by FaliseDotCom** [dannyvankooten/mailchimp-for-wordpress#89]
First name, last name, e-mail address texts appear in English eventhough the site's locale is Dutch. Texts are not translated in the nl_NL.po file. Required fields message (* = required) is translated so the translation seems to be incomplete.

**Comment by dannyvankooten** [comment]
Hi @FaliseDotCom ,

Did you maybe first create your form after switching your site language to Dutch? The form fields are translatable when you create your form.

Please re-add all your form fields using the "Add MailChimp field" tool located on the right side of the screen. You can then translate all parts (labels, placeholders, etc) into the language & wording of your choice.

Hope that helps. If not, let me know!

**Comment by FaliseDotCom** [comment]
He Danny

I created a list in Mailchimp, then installed the Wordpress Plugin, then
created a widget. The site's language / locale has always been set to
Dutch. I used this filter to translate the texts for now:

function _translate_mailchimp_texts($safe_text, $text )
{
switch ($safe_text)
{
case 'First Name':
return 'Voornaam';
case 'Last Name':
return 'Achternaam';
case 'Email Address':
return 'E-mail';
default:
return $safe_text;
}
}
add_filter( 'esc_html', '_translate_mailchimp_texts', 999, 2);

Btw, are you Dutch?


Met vriendelijke groet,
Sander Falise

*Falise.com*
*webdevelopment*
-----------------------------------
w : www.falise.com
e : sander@falise.com
t  : +31 (0) 6 - 230 22 467

2015-04-08 11:50 GMT+02:00 Danny van Kooten <notifications@github.com>:
